### PR TITLE
Ensure clean state after npm install

### PR DESCRIPTION
**Pull request by enocom**

### CODE REVIEW

**Comment by enocom** [comment]
This will fail until https://github.com/GoogleCloudPlatform/gcping/pull/79 is merged.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -41,3 +41,5 @@ jobs:
         cd web
         npm install
         npm run lint
+        echo "Checking for clean state..."
```
Comment (crwilcox) [inline .github/workflows/tests.yml:44]: This is to catch unexpected package-lock changes right? Could we add a comment here or more clear message to ensure we remember :)

Comment (enocom) [reply .github/workflows/tests.yml]: Done.

**Comment by enocom** [comment]
This isn't working on CI. Adding a "do not merge" tag for now.

**Pull request closed without merging**